packages/debian/README.md

```diff
@@ -0,0 +1,31 @@
+<h1>
```

Comment:
Is it needed?

Reply:
It's not mandatory but it can be useful for now.

Reply:
Let's not start littering the repo with `.md` files again. We have real documentation, let's use it please :)

Reply:
This is dev only and only for a very specific purpose though (building packages by hand).

Reply:
`doc/developer`

Reply:
The file will be kept until it is automated.
When it will be automated, the .md content file will be integrated in the documentation and will be deleted.
Does everyone is agree ?

Reply:
Agree. What we want to document is how to build package using a container +  `entrypoint.sh`

This is only temporary note until we're here.